src/widgets/AutocompleteSelector.js

```diff
@@ -93,12 +93,11 @@ export class AutocompleteSelector extends React.PureComponent {
       <View style={localStyles.container}>
         <SearchBar
```

Comment:
We also need to provide new styles. The old `SearchBar` was white, however the default for this one is orange - we are replacing the search bar, so we should not see any difference here - we need to make it white please!

Reply:
done.